DirectProgramming/DPC++/DenseLinearAlgebra/jacobi_iterative/sycl_dpct_migrated/src/jacobi.h

```diff
@@ -0,0 +1,12 @@
+//=========================================================
+// Copyright © 2020 Intel Corporation
```

Comment:
shouldn't it be 2022? same question for other files

```suggestion
// Copyright © 2022 Intel Corporation
```